pennylane/_device.py

```diff
@@ -213,9 +228,19 @@ def map_wires(self, wires):
 
     @classmethod
     def capabilities(cls):
-        """Get the other capabilities of the plugin.
+        """Get the capabilities of this device class.
+
+        Inheriting classes that change or add capabilities must override this method, for example via
 
-        Measurements, batching etc.
+        .. code-block::
+
+            @classmethod
+            def capabilities(cls):
+                capabilities = super().capabilities().copy()
+                capabilities.update(
+                    new_capability=...,
```

Comment:
Might be good to fill this in with a minimal example, so that it is copy and pastable